WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController.m

```diff
@@ -409,7 +411,7 @@ - (void)viewWillAppear:(BOOL)animated
 
     [self observeWillEnterForegroundNotification];
 
-    if (self.splitViewControllerIsHorizontallyCompact) {
+    if (!self.splitViewControllerIsHorizontallyCompact) {
```

Comment:
Doing this would mean that the selected index path won't be restored on iPad. Is this the intended behavior? I saw you added an isSplitViewDisplayed helper - maybe you can use that instead?
```suggestion
    if (self.isSplitViewDisplayed) {
```

Reply:
Good catch, thanks! 🙇